README.md

```diff
@@ -13,11 +13,15 @@ An env agnostic serializer and deserializer with recursion ability and types bey
 
 Serialized values can be safely stringified as *JSON* too, and deserialization resurrect all values, even recursive, or more complex than what *JSON* allows.
 
+### Installation
+`yarn add @ungap/structured-clone`
```

Comment:
can we *not* use *yarn* here as that's a variant of the de-facto standard in the industry, also the place where yarn grub code, which is *npm*?